### DISCUSSION TITLE
Build Error When Importing Scripts From ESM

### DISCUSSION THREAD

**Discussion by simonferndriger** [parcel-bundler/parcel#9170] [Q&A] [answered]
Hi

I get the following build error for my app.js which imports Alpine from esm.sh:

> 🚨 Build failed.
> @parcel/core: Failed to resolve 'https://esm.sh/alpinejs' from './dev/_new/script/app.js'
>
>  C:\Users\Simon\Documents\Adobe\Creative Cloud Files\Websites\swisswebcams.ch\dev\_new\script\app.js:4:30
>    3 | //--------------------------------------------------------------
>   4 | import Alpine           from 'https://esm.sh/alpinejs';
>      |                                 ^^^^^^^^^^^^^^^^^^^^^^^^^
>   5 |
>
> @parcel/resolver-default: Unknown url scheme or pipeline 'https:'

Is there another way to build this that works?

**Comment by mischnic** [comment, marked as answer]
See https://github.com/parcel-bundler/parcel/issues/4148
